### PR TITLE
Add rubygem-sentry-raven package

### PR DESCRIPTION
**Pull request by dgoetz**
This is a required dependency for the Foreman plugin foreman_concrete which will follow as a separate PR. I will also cherry-pick it for Foreman 1.23 afterwards.

### CODE REVIEW

**Comment by dgoetz** [comment]
I do not understand the error (including since when the spec is also in git annex). So if anyone has some advice I would be happy!

**Comment by ehelms** [comment]
The spec file appears to think its a git annex added file rather than the spec file itself. That is the first big issue its hitting.

**Comment by dgoetz** [comment]
I created a new system to create and push the changes and now it works fine, so it seems like an issue with my Fedora 30.

**Comment by dgoetz** [comment]
Rebased to resolve merge conflicts. Can someone do me the favor of a review? I want to get foreman_concrete packaged.

File: packages/plugins/rubygem-sentry-raven/rubygem-sentry-raven.spec [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+mkdir -p %{buildroot}%{_bindir}
+cp -a .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/exe -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/raven
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.gitmodules
```
Comment (ekohl) [inline packages/plugins/rubygem-sentry-raven/rubygem-sentry-raven.spec]: Is this one needed? It implies there's git submodules.

**Pull request merged**